ansible/files/namespace_throttlings_design_document_for_subjects_db.json

```diff
@@ -0,0 +1,10 @@
+{
+  "_id": "_design/namespaceThrottlings",
+  "views": {
+    "blockedNamespaces": {
+      "map": "function (doc) {\n  if (doc._id.indexOf(\"/limits\") >= 0) {\n    if (doc.concurrentInvocations === 0 || doc.invocationsPerMinute === 0) {\n      var namespace = doc._id.replace(\"/limits\", \"\");\n      emit(namespace, 1);\n    }\n  } else if (doc.subject && doc.namespaces && doc.blocked) {\n    doc.namespaces.forEach(function(namespace) {\n      emit(namespace.name, 1);\n    });\n  }\n}",
```

Comment:
is there anyway for the index of `/limits` to be `0`? ie should this be a strictly `> 0` test?

Reply:
I don't think so. Slashes are generally not allowed in namespaces so a namespace cannot possibly start with it.

The better check here would've been an `endWith` (ofcourse we'd need to emulate it through an indexOf + offset check). But we decided against that for improved readability.